node/crypto.ts

```diff
@@ -1,6 +1,103 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent, Inc. and Node.js contributors. All rights reserved. MIT license.
```

Comment:
What specifically requires the copyright attribution? I don't see anything that looks like it's lifted straight from node.

Reply:
I didn't copy the source code, but I coped some of the API documentation such as `class Hash`, `update`, `digest`, etc.